src/blog/2022/08/flowforge-08-released.md

```diff
@@ -0,0 +1,70 @@
+---
+title: FlowForge 0.8 released
+subtitle: Inter-Project Communication, Default Teams, and realtime device management.
+description: Inter-Project Communication, Default Teams, and realtime device management.
+date: 2022-08-04 12:00:00.0
+authors: ["sam-machin"]
+video: BBNzFInc_2E
+---
+TODO! Update Video ID<br>
+
+Easily pass messages between your projects on the cloud or devices, UX improvements, and more.
+
+<!--more-->
+
+Keep reading for  the details of whats in this release our you can watch our 1 min roundup video of the new release above. 
+
+We're pleased to announce version 0.8 is now available. the next release of the FlowForge application.
+
+## Features
+[Project Link Nodes](https://github.com/flowforge/flowforge/issues/662)
+We've introduced some custom FlowForge nodes to the pallette of new projects, These are Project Link nodes that allow you to easily pass data between different projects within the same team. 
+These projects can be running in the cloud or on devices, the communication is powered by our own internal MQTT broker.
+These nodes are available to all users of FlowForge Cloud but will only work with an Enterprise Edition of FlowForge.
+Try these out today on FlowForge Cloud by creating a new project or updating your stack, there's more information in the [README](https://github.com/flowforge/flowforge-nr-project-nodes/blob/main/README.md) for the nodes 
+![](../images/ProjectLink.gif)
+
+[Start & Stop Flows](https://github.com/flowforge/flowforge/issues/839)
+Node-RED 3.0 [introduced a new feature](https://nodered.org/blog/2022/07/14/version-3-0-released#editing-stopped-flows) that allows you to stop your flows from processing requests while still being able to work in the editor and deploy changes, We've now enabled this feature within FlowForge for projects running a Node-RED 3.x stack.
+
+[Default Team](https://github.com/flowforge/flowforge/issues/298)
+If you are a member of multiple teams you can now set your preferred default saving you from having to change teams each time you log in.
+
+## Improvements
+We've made a number of improvements to the overall experience of running FlowForge.
+
+- Devices now communicate to the Forge application over MQTT instead of polling [#754](https://github.com/flowforge/flowforge/issues/754)
```

Comment:
```suggestion
- Devices now communicate to the Forge application over MQTT instead of polling [#754](https://github.com/flowforge/flowforge/issues/754). You'll need to update your Device Agent to the latest version to take advantage of this.
```